flowise-embed/src/utils/fileClassificationUtils.ts

```diff
@@ -38,8 +38,8 @@ export const defaultChecklist = `
 • Nº de Série - (se mercadoria é máquina ou equipamento)
 • Informação Wooden Packing - (Not applicable; Treated and Certified; Not-Treated and Not-Certified; Processed)
 • NCM - (4 a 8 dígitos de cada NCM)
-• Valor unitário de cada espécie de mercadoria
-• Valor total de cada espécie de mercadoria
+• Valor unitário de cada espécie de mercadoria - Considerando todas as páginas é o valor unitário multiplicado pela quantidade das mercadorias(quantidade X valor unitário, separando mercadorias diferentes por ';')
+• Valor Total de cada espécie de mercadoria - Considerando todas as páginas é o valor unitário multiplicado pela quantidade das mercadorias ou apenas o valor total já informado no documento (quantidade X valor unitário separando mercadorias diferentes por ';')
```

Comment:
Além da vírgula que o @matheusmondaini-startse sugeriu, um espaço antes de abrir parênteses é bem-vindo também.
```suggestion
• Valor unitário de cada espécie de mercadoria - Considerando todas as páginas, é o valor unitário multiplicado pela quantidade das mercadorias (quantidade X valor unitário, separando mercadorias diferentes por ';')
• Valor Total de cada espécie de mercadoria - Considerando todas as páginas, é o valor unitário multiplicado pela quantidade das mercadorias ou apenas o valor total já informado no documento (quantidade X valor unitário separando mercadorias diferentes por ';')